todo_app/data/trello_items.py

```diff
@@ -0,0 +1,111 @@
+import json
+from os import getenv
+from flask import session
+import requests
+
+from todo_app.domain.item import Item
+
+# use env var for prefix
+cards_url = "https://api.trello.com/1/cards"
+board_url = "https://api.trello.com/1/boards/65c4fed365273e251af75aa5"
+todo_list_id = "65c4fed42c4a7016163343f9"
+
+headers = {
+    "Accept": "application/json"
+}
+query = {
+    'key': getenv('TRELLO_API_KEY'),
+    'token': getenv('TRELLO_API_TOKEN')
+}
+
+def get_list_organised_items():
+    """
+    Fetches all saved items for each status in the trello board.
+
+    Returns:
+        list: The list of list names with corresponding items.
+    """
+    items_response = requests.request(
+        "GET",
+        f"{board_url}/cards",
+        headers=headers,
+        params=query
+    )
+    items = json.loads(items_response.text)
+
+    lists_response = requests.request(
+        "GET",
+        f"{board_url}/lists",
+        headers=headers,
+        params=query
+    )
+    list_organised_items = [{
+        "name": list["name"],
+        "entries": [Item.from_trello_card(item, list) for item in items if item["idList"] == list["id"]]
+        } for list in json.loads(lists_response.text) ]
+
+    return list_organised_items
+
+def get_lists():
+    """
+    Fetch all lists in board
+
+    Returns:
+        list: list of lists in the board
+    """
+    lists_response = requests.request(
+        "GET",
+        f"{board_url}/lists",
+        headers=headers,
+        params=query
+    )
+    
+    return json.loads(lists_response.text)
```

Comment:
I'm not convinced I have a real preference, but worth noting we can equally use:
```python
    return lists_response.json()
```